sros2/test/sros2/commands/security/verbs/test_create_key.py

```diff
@@ -34,19 +34,20 @@
 
 # This fixture will run once for the entire module (as opposed to once per test)
 @pytest.fixture(scope='module')
-def node_keys_dir(tmpdir_factory):
-    keystore_dir = str(tmpdir_factory.mktemp('keystore'))
+def security_context_keys_dir(tmpdir_factory):
+    keystore_dir = Path(str(tmpdir_factory.mktemp('keystore')))
```

Comment:
Has pytest been updated enough we can use `tmp_path_factory` here now? Would save you a little work.

Reply:
Sure, but I think we can left that for a follow-up, because I'm not sure which `pytest` version we're using in the different platforms.